File: content/snippets/typescript/language_basics/dates_and_time.ts
#!/usr/bin/env ts-node

import { setImmediate } from "node:timers";

function main(): void {
  const now = new Date();
  const later = new Date(now.getTime() + 90 * 60 * 1000);

  console.log("now:", now.toISOString());
  console.log("later:", later.toISOString());
  console.log("later > now:", later > now);
}

main();
setImmediate(() => {});

File: content/snippets/python/language_basics/dates_and_time.py
#!/usr/bin/env python3
from datetime import datetime, timedelta


def main() -> None:
    now = datetime.now()
    later = now + timedelta(minutes=90)

    print("now:", now.isoformat(timespec="seconds"))
    print("later:", later.isoformat(timespec="seconds"))
    print("later > now:", later > now)


if __name__ == "__main__":
    main()
